Check and clean orchestrator branch (#16)
lifebit-ai/end-to-end-target-identification: bin/report/report.Rmd

# Step 3. Identifying Causal Genes and Pathways

This step can be used to prioritise molecular phenotypes (like expression level of a gene, DNA methylation level at CpG sites) that mediate the genetic effects for complex trait and thus provide mechanistic interpretation of the GWAS signal.

# Step 5. Identifying Mechanism of Action

This pipeline uses an enrichment tool to identify the associations between variants and tissues. The plot below describe the p-value associated with the SNPs enrichment for each tissue

> The tables below describe instead the data used for the above plot and the output of finemapping in case this input is used.

Reviewer: ```suggestion
The table below contains the data used for the plot.
```
